CONTRIBUTING.md

```diff
@@ -62,34 +68,46 @@ Finally, for _development dependencies only_, you must add an entry to
 `ci/environment-dev.yaml` with the same version constraint as in
 `pyproject.toml`.
 
-## First Steps to fix an issue or bug
+## First Steps to contribute
+
+- Read the documentation
+- Fork this repo (see "Development environment" section above for more)
+- Install environment (see "Development environment" section above for more)
+- Run the unit tests successfully in `main` branch:
+    - `make test`
+
+### ...to fix an issue or bug
 
-- Read the documentation (working on adding more)
-- create the minimally reproducible issue
-- try to edit the relevant code and see if it fixes it
-- submit the fix to the problem as a pull request
-- include an explanation of what you did and why
+- Create a GitHub issue with a minimally reproducible example (TODO: Link to SSCCE?)
```

Comment:
Should this be a draft PR until you address this TODO?

Reply:
For sure for this one. There's "competitors" to sscce.org, do you have strong feelings about any of them? I figured I'd google around and link to whatever seems most popular.

Reply:
Alright, picking one has been less than successful. I vote we use the term ["minimal reproducible example"](https://en.wikipedia.org/wiki/Minimal_reproducible_example) and consider all these other things to be "brand names" of effectively the same thing. Choosing one seems to be folly. We can link to the wikipedia page and let people follow through to more resources?

* MCVE: [Stack overflow has an excellent page MCVE page](https://stackoverflow.com/help/minimal-reproducible-example), but I'm not certain if SO is the originator or what.
* SSCCE: Has an official <http://www.sscce.org> website. If you hit it with HTTPS by accident, the site doesn't work! Abbreviation is hard to remember.
* reprex: I think of <https://reprex.tidyverse.org/> as the "official site". The term somehow feels more jargony to me than an acronym :laughing: Just a feeling though.

Reply:
6fcaacd